config/dictionaries/resource.json

```diff
@@ -661,42 +683,111 @@
         "summary": "Remove a fleet service variable",
         "description": "",
         "method": "DELETE",
-        "endpoint": "/v5/service_environment_variable(<ID>)",
+        "endpoint": "/v6/service_environment_variable(<ID>)",
         "filters": ""
       }
     ]
   },
   {
-    "id": "release",
-    "name": "Release",
+    "id": "organization",
+    "name": "Organization",
     "fields": [
       "id",
       "created_at",
-      "belongs_to__application",
-      "is_created_by__user",
-      "commit",
-      "composition",
-      "status",
-      "source",
-      "start_timestamp",
-      "end_timestamp",
-      "update_timestamp"
+      "name",
+      "handle",
+      "company_name",
+      "website",
+      "industry",
+      "billing_account_code"
     ],
+    "examples": [
+      {
+        "id": "get-all-orgs",
+        "summary": "Get all organizations associated with authenticated user",
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/organization",
+        "filters": ""
+    },
+    {
+        "id": "org-by-id",
+        "summary": "Get organization by ID",
+        "description": "",
+        "method": "GET",
+        "endpoint": "/v6/organization(<ID>)",
+        "filters": ""
+      },
+      {
+        "id": "org-by-handle",
+        "summary": "Get organization by handle",
+        "description": "Handles are a simpler way to identify organizations. For example: An organization called Foo Bar Enterprises will have the handle foo_bar_enterprises. For gh_john_doe's Organization the handle will be gh_john_doe.",
```

Comment:
```suggestion
        "description": "Uniquely identifies an organization. Handles are currently automatically generated and read-only.",
```

Reply:
This was marked as resolved I suspect accidentally.

Reply:
My bad, didn't push the commit fixing this bit. Coming right up.